ui/component/or-map/src/markers/or-map-marker-asset.ts

```diff
@@ -1,28 +1,18 @@
 import {PropertyValues} from "lit";
 import {customElement, property} from "lit/decorators.js";
 import {OrMapMarker} from "./or-map-marker";
-import {AttributeEvent, GeoJSONPoint, AssetEvent, AssetEventCause, Asset, SharedEvent, AssetDescriptor, WellknownAttributes} from "@openremote/model";
```

Comment:
Re-formatting mixed in with code changes is a big headache to figure out what are changes and what's formatting. We need to put some standard formatting rules in place but for now please don't re-format whole files at the same time as making change.